src/tests/JIT/Regression/JitBlue/Runtime_34587/Runtime_34587.cs

```diff
@@ -80,7 +81,9 @@ public static int Main()
         bool succeeded = true;
 
         succeeded &= ValidateArm();
+        Console.WriteLine($"ValidateArm:{succeeded}");
```

Comment:
nit: This will only correctly report the first failure. If say, `Arm` fails it will also report all subsequent validations as failing.

Reply:
nit: If you fix this, might be nice to also add a space between `:` and `{succeeded}` to help with readability